curl-helper.c

```diff
@@ -5377,6 +5377,100 @@ value caml_curl_check_enums(value v_unit)
   CAMLreturn(v_r);
 }
 
+/* WebSocket flags mapping: same order as OCaml variant */
+static const long wsFlags[] = {
```

Comment:
all the code in this PR should be under one HAVE_DECL_CURL_WS_META because all these were added at once and no need to check separately except for CURLWS_PONG and OFFSET need separate check because they were added later afaics (can do #ifndef CURLWS_PONG then 0 in code)